events2.js

```diff
@@ -0,0 +1,63 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+    let form = document.querySelector(".form");
+    let button = document.querySelector("button")
+
+
+    form.addEventListener("submit", event => {
+      event.preventDefault();
+      // console.log(event, textInput);
+      let input = document.querySelector(".input");
+      let textInput = input.value;
+      let ul = document.querySelector("ul");
+
+      if (textInput) {
+        let newLi = document.createElement("li");
+        let newButton = document.createElement("button");
+        newLi.innerText = textInput;
+        newButton.innerText = "Remove";
+        ul.appendChild(newLi);
+        ul.appendChild(newButton)
+
+
+        newLi.addEventListener("click", event => {
+
+          if (event.target.style.textDecoration !== "line-through"){
+            event.target.style.textDecoration = "line-through"
+          } else {
+            event.target.style.textDecoration = "none"
+          }
+        })
+
+        newButton.addEventListener("click", event => {
+          if (newLi.style.textDecoration !== "line-through") {
+            alert("First cross out this To-Do.")
+          } else
+          if (newLi.style.textDecoration === "line-through") {
+            newLi.parentNode.removeChild(newLi)
+            newButton.parentNode.removeChild(newButton)
+          }
+        })
+
+      }else {
+        let newP = document.createElement("p");
+        // console.log(newP);
+        newP.innerText = "Error. You didn't add a To-Do.";
+        form.appendChild(newP)
+      }
```

Comment:
To avoid appending a new error message every time an invalid input is attempting to be submitted and added to our form, you could make the following changes:

- Instead of creating the new `p` element here, you could create it earlier (ex: on line 5)
- In your `else` block on line 42, you need only change the innerText of the `p` element to the text of your choice

```
} else {
    newP.innerText = "Error. You didn't add a To-Do.";
}
```